contracts/shared/Structs.sol

```diff
@@ -70,7 +70,7 @@ contract Structs {
         address issuer;
         address paymentMethod;
```

Comment:
Can we rename it to something like `paymentToken` or `paymentTokenAddress` or something else? 
as word **method** tells "how"  something is paid (e.g. cash, card, "u naturi").

Reply:
done